src/sardana/spock/spockms.py

```diff
@@ -290,7 +292,7 @@ def __init__(self, name, **kw):
         self.call__init__(BaseDoor, name, **kw)
 
     def create_input_handler(self):
-        return SpockInputHandler(self)
+        return SpockInputHandler()
```

Comment:
Good spotting!

Reply:
Thanks - I caught it when I was testing it.